integrations/openai/src/index.ts

```diff
@@ -288,7 +290,25 @@ export default new bp.Integration({
           defaultModel: DEFAULT_LANGUAGE_MODEL_ID,
           overrideRequest: (request) => {
             if (input.model?.id.startsWith('o1-') || input.model?.id.startsWith('o3-')) {
-              request.reasoning_effort = input.reasoningEffort
+              if (input.reasoningEffort === 'none') {
+                const acceptedValues = SupportedReasoningEfforts.map((x) => `"${x}"`)
+                  .map((x, i) => (i === SupportedReasoningEfforts.length - 1 ? `or ${x}` : x))
```

Comment:
Am I wrong? seems like the output would be:
`low, medium, or high`

is this what we want?